backend/bingo/src/generate.rs

```diff
@@ -0,0 +1,127 @@
+use rand::rngs::StdRng;
+use rand::Rng;
+
+use crate::matrix::Transpose;
+
+/// min以上max以下の数字をランダムに返す
+/// 実行する度に違う値が返ります
+///
+/// # 例
+///
+/// ```
+/// # use bingo::generate::number;
+/// let mut rng = rand::SeedableRng::seed_from_u64(0);
+/// assert_eq!(number(&mut rng, 1, 75), 59);
+/// assert_eq!(number(&mut rng, 1, 75), 17);
+/// ```
+pub fn number(rng: &mut StdRng, min: usize, max: usize) -> usize {
+    rng.gen_range(min..=max)
+}
+
+/// sizeで与えられた大きさの数字盤(二次元配列)を返す
+/// ```
+/// # use bingo::generate::board_numbers;
+/// let mut rng = rand::SeedableRng::seed_from_u64(0);
+/// assert_eq!(board_numbers(&mut rng, 3), vec![[12, 19, 34], [1, 0, 42], [9, 27, 45]]);
+/// ```
+///
+/// sizeに偶数が渡された場合，panicする
+/// ```should_panic
+/// # use bingo::generate::board_numbers;
+/// let mut rng = rand::SeedableRng::seed_from_u64(0);
+/// board_numbers(&mut rng, 2);
+/// ```
+pub fn board_numbers(rng: &mut StdRng, size: usize) -> Vec<Vec<usize>> {
+    if size % 2 == 0 {
+        // TODO: 要件が増えたときに直す
+        panic!("size should be odd number");
+    }
+
+    let mut board: Vec<Vec<usize>> = vec![];
+    for row in 0..size {
+        let min = row * 15 + 1;
+        let max = (row + 1) * 15;
+        let mut column = vec![];
+
+        while column.len() < size {
+            // フリーマス
+            if row == size / 2 && column.len() == size / 2 {
+                column.push(0)
+            }
+
+            let gen_number = number(rng, min, max);
+
+            if !column.iter().any(|&x| x == gen_number) {
+                column.push(gen_number)
+            }
+        }
+
+        board.push(column)
+    }
+
+    board.into_iter().transpose().collect()
+}
+
+#[cfg(test)]
+mod tests {
+    use super::*;
+
+    #[test]
+    fn generate_number() {
```

Comment:
```suggestion
    fn it_can_generate_numbers() {
```

⚪︎⚪︎ができる！みたいなテスト名にして欲しいです

Reply:
他のテストケースも同様に対処お願いします